agent/engine/serviceconnect/manager_linux.go

```diff
@@ -53,6 +53,12 @@ const (
 	defaultStatusFileName     = "appnet_admin.sock"
 	defaultStatusENV          = "APPNET_AGENT_ADMIN_UDS_PATH"
 
+	// logging
+	defaultLogPathHostRoot              = "/var/log/ecs/service_connect/"
```

Comment:
Could `/var/log` be sourced from a config variable somewhere? I believe this path is changeable, and is changed for example when ecs-init is built with `make dev` with the "directory prefix" var in ecs-init: 

prefix set to /tmp here: https://github.com/aws/amazon-ecs-agent/blob/71a0f6765cc469913a0b09728196feb0d0d33e37/ecs-init/config/development.go#L36


sourced here: https://github.com/aws/amazon-ecs-agent/blob/637446af461d77e75c4556cd55e6771bd4250fe4/ecs-init/config/common.go#L155

Reply:
Discussed offline - it's quite challeging to get the host path of log directory from inside ECS Agent. We would need to get the host path from Agent container bind mount info, for which we need a docker inspect and agent container ID, which in turn probably needs a docker ls with image name filter. 

The risk of possible impact of log host dir being something other than `/var/log/ecs/` should be minimal - most customers run ECS Agent with init which defaults to `/var/log/ecs`. Even if the log path on instance is explicitly updated by specifying a different volume mount, and as a result we fail to configure the correct log dir for AppNet, it will be able to fall back to the stdout/stderr and logs will be available through docker log json file. Other than that, AppNet should continue to function properly.